spark/pyspark/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.7.3-stretch
+
+ARG BAZEL_REPO="deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8"
+ARG BAZEL_RELEASE_KEY_URL="https://bazel.build/bazel-release.pub.gpg"
+
+ARG DOCKER_REPO="deb [arch=amd64] https://download.docker.com/linux/debian stretch stable"
+ARG DOCKER_RELEASE_KEY_URL="https://download.docker.com/linux/debian/gpg"
+
+RUN apt-get update && apt-get install -y apt-transport-https
```

Comment:
```suggestion
RUN apt-get install -y apt-transport-https
```

Reply:
Unfortunately:

```
docker build . -t pubnative/pyspark-ci:1.0.2
Sending build context to Docker daemon  4.096kB
Step 1/10 : FROM python:3.7.3-stretch
 ---> 59a8c21b72d4
Step 2/10 : ARG BAZEL_REPO="deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8"
 ---> Using cache
 ---> cc4cc0d826a4
Step 3/10 : ARG BAZEL_RELEASE_KEY_URL="https://bazel.build/bazel-release.pub.gpg"
 ---> Using cache
 ---> c99b997388ef
Step 4/10 : ARG DOCKER_REPO="deb [arch=amd64] https://download.docker.com/linux/debian stretch stable"
 ---> Using cache
 ---> 8e8ceeba2ac6
Step 5/10 : ARG DOCKER_RELEASE_KEY_URL="https://download.docker.com/linux/debian/gpg"
 ---> Using cache
 ---> 0b542e16d1bd
Step 6/10 : RUN apt-get install -y apt-transport-https
 ---> Running in 6d9c06d1a177
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package apt-transport-https
The command '/bin/sh -c apt-get install -y apt-transport-https' returned a non-zero code: 100
make: *** [build] Error 100
```

Reply:
I had already tried, but it fails if there are no updates.

Reply:
i see. good to know.